web/pageComponents/shared/Header.tsx

```diff
@@ -180,12 +180,14 @@ const Header = ({ slugs, menuData }: HeaderProps) => {
                     }}
                     style={
                       {
+                        '--eds_button__padding_x': 'var(--space-small)',
                         color: 'var(--default-text)',
                         fill: 'var(--default-text)',
                       } as CSSProperties
                     }
                   >
                     <Icon size={24} data={search} />
+                    <FormattedMessage id="search"></FormattedMessage>
```

Comment:
Is it possible to make it self-closed instead? 
`<FormattedMessage id="search" />`